Bottom: 937052cdd60bb8b9a4cbc05ad1abcf257dd22ff0
Top:    11bb2332dca63a99d9328b312f59e866286188ad
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 09:42:37 -0500

Point python tools to internal pypi mirror



---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg b/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
index fb6f7f4..8dc2a29 100644
--- a/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
+++ b/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
@@ -1,2 +1,2 @@
 [easy_install]
-index-url=https://pypi.python.org/simple
+index-url=http://172.16.103.72/repository/PyPi_Proxy/simple
